### PR TITLE
Fix default value for ratings

### PR DESCRIPTION
**Pull request by hedudelgado**
@ouranos @alexnoox

### CODE REVIEW

File: src/app/views/marketplace/modals/create-review-modal.controller.coffee [hunk 1]
```diff
@@ -5,6 +5,7 @@ angular.module 'mnoEnterpriseAngular'
   vm.modal = {model: {}}
   vm.appRating = 5
   vm.app = {}
+  vm.hasClicked = $scope.hasClicked
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.controller.coffee]: I think we should remove the `$scope` injection and only use `vm`.

Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.controller.coffee]: You can delete this line then.

Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.controller.coffee]: Line 26: 
replace: `$scope.modal.isLoading = false` by `vm.modal.isLoading = false`

File: src/app/views/marketplace/modals/create-review-modal.html [hunk 2]
```diff
@@ -6,15 +6,18 @@
   <!-- Modal Body -->
   <div class="modal-body">
     <div class="pull-right">
-      <star-rating ng-model="vm.appRating"></star-rating><br/>
+      <star-rating ng-model="vm.appRating" has-clicked="hasClicked"></star-rating><br/>
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.html]: Use `vm. hasClicked` here.

File: src/app/views/marketplace/modals/create-review-modal.html [hunk 3]
```diff
@@ -6,15 +6,18 @@
...
+    </div>
+    <div class="pull-right" ng-if="!hasClicked">
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.html]: Use `vm.hasClicked` here.

File: src/app/views/marketplace/modals/create-review-modal.html [hunk 4]
```diff
@@ -6,15 +6,18 @@
...
     </div>
     <textarea class="form-control" rows="5" ng-model="vm.modal.model.description" placeholder="{{ 'mno_enterprise.templates.dashboard.marketplace.show.tell_others' | translate}}" id="comment"></textarea>
   </div>
   <div class="modal-footer">
     <div class="row">
       <div class="col-sm-12">
         <button class="btn btn-default" ng-click="vm.modal.cancel()" ng-show="!vm.modal.isLoading">{{ 'mno_enterprise.templates.dashboard.marketplace.show.cancel' | translate}}</button>
-        <button class='btn btn-info' ng-click="vm.modal.proceed()" ng-disabled="vm.modal.isLoading || form.$invalid">
+        <button class='btn btn-info' ng-click="vm.modal.proceed()" ng-disabled="vm.modal.isLoading || form.$invalid || !hasClicked">
           <span ng-show="vm.modal.isLoading"><i class="fa fa-spinner fa-pulse fa-fw"></i></span>
```
Comment (alexnoox) [inline src/app/views/marketplace/modals/create-review-modal.html]: Use `vm.hasClicked` here.

**Comment by hedudelgado** [comment]
@alexnoox

**Comment by alexnoox** [comment]
I reimplemented this feature in commit https://github.com/maestrano/mno-enterprise-angular/pull/106/commits/4a741b7ce1ece0af0dd5517182b1595f8ff9b509 in PR https://github.com/maestrano/mno-enterprise-angular/pull/106

**Pull request closed without merging**